index.js

```diff
@@ -22,6 +22,36 @@ const TREZOR_CONNECT_MANIFEST = {
   appUrl: 'https://metamask.io',
 };
 
+const oneKeySpecialVersion = 99;
+const oneKeyVendor = 'onekey.so';
+
+/**
+ * get the vendor name of the hardware wallet
+ * @param {object} features
+ * @returns {'onekey' | 'trezor'}
+ */
+const getVendorName = (features) => {
```

Comment:
Looks like we use `function` in this file to define functions instead of `const`, do you mind updating this to keep the style consistent?

Reply:
Yes, we should keep the code style consistent. 👍